pkgs/by-name/st/streamcontroller/package.nix

```diff
@@ -16,18 +16,24 @@
 stdenv.mkDerivation rec {
   pname = "streamcontroller";
 
-  version = "1.5.0-beta.7";
+  version = "1.5.0-beta.8";
   # We have to hardcode revision because upstream often create multiple releases for the same version number.
-  # This is the commit hash that maps to 1.5.0-beta.7 released on 2024-11-20
-  rev = "45b5bc72f617c5aea306450d6592da66ade53568";
+  # This is the commit hash that maps to 1.5.0-beta.8 released on 2025-03-12
+  rev = "de11d84afac7873044568606a8468c78d57aceda";
 
   src = fetchFromGitHub {
     repo = "StreamController";
     owner = "StreamController";
     inherit rev;
-    hash = "sha256-tgbqURtqp1KbzOfXo4b4Dp3N8Sg8xcUSTwdEFXq+f6w=";
+    hash = "sha256-pE92/oX9iZYCIhwDkPkjPq/cDUQLUGs+Ou5rjFEIBpo=";
   };
 
+  # Some plugins needs to load things dynamically and in that case we won't find python3 without this
```

Comment:
Looks good. if we find ourselves patching multiple source files for this issue in the future, maybe we can consider simply adding `python3` to `PATH` env var

Reply:
`python3` might be good to add, but currently it's not too bad. I've made a local patch for `kdotool` as well but that's only needed for KDE so not sure how to handle that.

Reply:
We either need a module or add it to the package for everyone.

Reply:
There is a module, should we move things like this there instead? And how do I know if the user is running KDE? Do I just look for `services.desktopManager.plasma6.enable = true;` ?

Reply:
I did this change in the module for kdotool, do we want to give the same treatment to python3?